app/code/Magento/Sales/Controller/Adminhtml/Order/Invoice/AddComment.php

```diff
@@ -39,29 +44,38 @@ class AddComment extends \Magento\Sales\Controller\Adminhtml\Invoice\AbstractInv
      */
     protected $resultRawFactory;
 
+    /**
+     * @var InvoiceRepositoryInterface
+     */
+    protected $invoiceRepository;
```

Comment:
Please for newly declared properties use only private access level

> 2.7. All non-public properties and methods SHOULD be private.

See [Magento Technical guideline](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html#2-class-design) 2.7 topic

Reply:
Done, thank you.